src/Markdown.svelte

```diff
@@ -59,6 +61,22 @@
         });
       }
     }
+
+    // Replaces code blocks in the background with highlighted code.
+    const prefix = "language-";
+    const nodes = Array.from(document.body.querySelectorAll("pre code"));
+
+    for (const node of nodes) {
```

Comment:
Wonder if it's worth parallelising this with something like `Promise.allSettled`?

Reply:
hmm yeah that makes sense :+1:

Reply:
fc1ba407